bls/BLSPrivateKeyShare.cpp

```diff
@@ -128,8 +128,7 @@ BLSPrivateKeyShare::generateSampleKeys( size_t _requiredSigners, size_t _totalSi
     std::vector< libff::alt_bn128_Fr > skeys = dkg_obj.SecretKeyContribution( pol );
 
     libff::alt_bn128_Fr common_skey = pol.at( 0 );
-    std::shared_ptr< BLSPublicKey > pkey_ptr =
-        std::make_shared< BLSPublicKey >( common_skey, _requiredSigners, _totalSigners );
+    std::shared_ptr< BLSPublicKey > pkey_ptr = std::make_shared< BLSPublicKey >( common_skey );
```

Comment:
please do `make bls-fromat` in your build directory and then push changes to your branch. we have a format-check for all the code pushed to develop so your build may fail if you don't do it.

Reply:
great, thank you!

Reply:
I actually already did `make bls-format` with all my commits. Is there still any piece of code that is not correctly formatted?

Reply:
then it should be good, thanks

Reply:
The only remaining issue is that with some commits, the ci/cd pipeline fails with this error in one of the test-cases, which is unrelated to my changes. Sometimes this test fails with the linux build, sometimes with the os x build and sometimes it just works fine.
![Screenshot 2021-08-24 at 16 48 02](https://user-images.githubusercontent.com/20229219/130638447-a2275f1e-d161-4c50-9ef9-f41b48022596.png)